xla/core/collectives/clique_id.h

```diff
@@ -59,6 +59,31 @@ H AbslHashValue(H h, const CliqueId& id) {
   return H::combine(std::move(h), id.data());
 }
 
+// Collection of CliqueIds
```

Comment:
Can you add a little bit more details what it really means for communicator creating, had to go to NCCL documentation to get the idea